source/index.html.md

```diff
@@ -50,18 +50,12 @@ This redirection will also contain the previously passed `state` value, as a que
 
 ### Step 3: Exchange the authorization code for an access token
 
-Make a `POST` request to `https://auth.grooveapp.com/oauth/authorize/token`
+Make a `POST` request to `https://auth.grooveapp.com/oauth/token`
 
 ```shell
-curl -X POST "https://auth.grooveapp.com/oauth/authorize"
-  -H 'Content-Type: application/json'
-  -d '{
-    "grant_type": "authorization_code",
-    "code": "SomeAuthorizationCode",
-    "redirect_url": "SomeRedirectUri",
-    "client_id": "SomeClientId",
-    "client_secret": "SomeClientSecret"
-  }'
+curl -X POST "https://auth.grooveapp.com/oauth/token"
+  -H "Content-Type: application/x-www-form-urlencoded"
+  -d "grant_type=authorization_code&code=SomeAuthorizationCode&redirect_uri=SomeRedirectUri&client_id=SomeClientId&client_secret=SomeClientSecret"
```

Comment:
I can split this up on new lines if you think this is too long

Reply:
I think that works, it depends on how it renders in the webpage

Reply:
Cool, you can check the screenshot in the PR description

Reply:
I think that's ok because that looks like one long string and splitting it up in multiple lines could introduce errors if for example the user adds a space at the end of one line